web/src/App.tsx

```diff
@@ -9,6 +10,7 @@ const App = () => {
         <main className="py-3">
           <div className="max-w-6xl mx-auto px-4">
             <Outlet />
+            <Exec/>
```

Comment:
Hey nice stuff! You've got the hang of using components, and I think for now this good. But in the future we would not directly have this in the app but have it in a different route like `/execs`! That is done through setting up routes which I will do but just for future reference :)